perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -388,25 +391,24 @@ def Run(benchmark_spec):
     for k, v in vm.GetMachineTypeDict().iteritems():
       metadata['{0}_{1}'.format(vm_specifier, k)] = v
 
-  num_streams = FLAGS.netperf_num_streams
-  assert(num_streams >= 1)
-
-  for netperf_benchmark in FLAGS.netperf_benchmarks:
-
-    if vm_util.ShouldRunOnExternalIpAddress():
-      external_ip_results = RunNetperf(client_vm, netperf_benchmark,
-                                       server_vm.ip_address, num_streams)
-      for external_ip_result in external_ip_results:
-        external_ip_result.metadata.update(metadata)
-      results.extend(external_ip_results)
-
-    if vm_util.ShouldRunOnInternalIpAddress(client_vm, server_vm):
-      internal_ip_results = RunNetperf(client_vm, netperf_benchmark,
-                                       server_vm.internal_ip, num_streams)
-      for internal_ip_result in internal_ip_results:
-        internal_ip_result.metadata.update(metadata)
-        internal_ip_result.metadata['ip_type'] = 'internal'
-      results.extend(internal_ip_results)
+  for num_streams in FLAGS.netperf_num_streams:
+    assert(num_streams >= 1)
+
+    for netperf_benchmark in FLAGS.netperf_benchmarks:
```

Comment:
Yes - this makes it a lot easier to run with several different stream counts - otherwise we'd have to provision and tear down vms for each of the netperf_benchmark tests.